draft-salowey-wimse-arch.md

```diff
@@ -55,10 +56,44 @@ TODO Introduction
 
 # Use Cases
 
+1. Basic Service Authentication
+
+One of the most basic use cases for workload identity is for authenticating one workload to another such as in the case where one microservice is making a request of another microservice within a larger application. Even in this simple case the identity of the workload is often a composite of many attributes such as:
```

Comment:
Does not have to be "micro"service. I would suggest generalising this to any service.

Reply:
Agree